packages/core/src/odata-v4/request-builder/get-all-request-builder.spec.ts

```diff
@@ -52,6 +53,17 @@ describe('GetAllRequestBuilder', () => {
     });
   });
 
+  it('is built correctly for selects inside of an expand.', async () => {
```

Comment:
I will go ahead and approve already, but consider whether this can be still phrased differently as have discussed in our "test strategy" meeting, that "is built correctly" is not very speaking. I can see that there are many tests like this in this file, so I wouldn't be too strict about this.